### PR TITLE
Character Warnings Extension

### LINKED ISSUE #181: Content Warning thumbnail for Characters
### Feature Request

It has recently come to my attention that the Gallery got a specific thumbnail for content warnings.
I feel it would be a good idea to extend this to the character masterlist.

### PR DESCRIPTION
**Pull request by ScuffedNewt**
Discussed with @SpeedyD to open a new PR for this since I went a slightly different approach to merging

Adds the aforementioned extension to dev
tested locally, todos sorted, only thing to not:
- in `BrowseController`, the `$filterImageQuery` variable is renamed in comparison to pr #876, and I also took a different approach (no `else`s), please review this part carefully
- in `_masterlist_content` I added the `['all' => 'include / exclude all']` in the form field (having both as exclude didn't make sense)

### CODE REVIEW

**Review by SpeedyD — APPROVED** [review]
Me? Forgetting to approve a PR? It's more likely than you think . ..

File: resources/views/browse/_masterlist_content.blade.php [hunk 1]
```diff
@@ -96,6 +96,18 @@
                 {!! Form::label('Has Traits: ') !!} {!! add_help('This will narrow the search to characters that have ALL of the selected traits at the same time.') !!}
                 {!! Form::select('feature_ids[]', $features, Request::get('feature_ids'), ['class' => 'form-control feature-select userselectize', 'placeholder' => 'Select Traits', 'multiple']) !!}
             </div>
+            @if (!$isMyo)
+                <div class="row">
+                    <div class="col-md-6 form-group">
+                        {!! Form::label('Exclude Selected Tags: ') !!} {!! add_help('This will exclude characters that have ANY of the selected traits at the same time.') !!}
```
Comment (itinerare) [inline resources/views/browse/_masterlist_content.blade.php:102]: I like this tactic of include vs exclude as separate filters... it's more straightforward to reason around
That said, help text needs to be updated for both fields!

Comment (ScuffedNewt) [reply resources/views/browse/_masterlist_content.blade.php:102]: ah yep, was thinking I'd forget something small like that
will update this after work

Comment (itinerare) [reply resources/views/browse/_masterlist_content.blade.php]: There's always something |D It can't be helped...

**Review by itinerare — APPROVED** [review]

**Pull request merged**